pkg/storkctl/clusterpair_test.go

```diff
@@ -0,0 +1,49 @@
+// +build unittest
+
+package storkctl
+
+import (
+	"reflect"
+	"testing"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestGetClusterPairNoPair(t *testing.T) {
+	cmdArgs := []string{"clusterpair"}
+
+	var clusterPairs storkv1.ClusterPairList
+	expected := "No resources found.\n"
+	testCommon(t, newGetCommand, cmdArgs, &clusterPairs, expected, false)
+}
+
+func TestGetClusterPairNotFound(t *testing.T) {
+	cmdArgs := []string{"clusterpair", "pair1"}
+
+	var clusterPairs storkv1.ClusterPairList
+	expected := `Error from server (NotFound): clusterpairs.stork.libopenstorage.org "pair1" not found`
+	testCommon(t, newGetCommand, cmdArgs, &clusterPairs, expected, true)
+}
+
+func TestGenerateClusterPair(t *testing.T) {
+	cmdArgs := []string{"clusterpair", "pair1"}
+
+	var clusterPairs storkv1.ClusterPairList
+	clusterPair := &storkv1.ClusterPair{
+		TypeMeta: meta.TypeMeta{
+			Kind:       reflect.TypeOf(storkv1.ClusterPair{}).Name(),
+			APIVersion: storkv1.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: meta.ObjectMeta{},
+
+		Spec: storkv1.ClusterPairSpec{
+			Options: map[string]string{},
+		},
+	}
+
+	clusterPairs.Items = append(clusterPairs.Items, *clusterPair)
+
+	expected := ""
```

Comment:
This should expect the table output, right?

Reply:
I think in my current test `config.Contexts[currentContext]` is nil, so it skips over a bunch of stuff in `newGenerateClusterPairCommand` and doesn't emit any output.  This is not ideal, but I couldn't figure out how to populate the currentContext